x-pack/plugins/security_solution/server/endpoint/services/actions/clients/crowdstrike/crowdstrike_actions_client.ts

```diff
@@ -330,6 +306,41 @@ export class CrowdstrikeActionsClient extends ResponseActionsClientImpl {
     return this.fetchActionDetails(actionRequestDoc.EndpointActions.action_id);
   }
 
+  private async completeCrowdstrikeAction(
+    actionResponse: ActionTypeExecutorResult<CrowdstrikeBaseApiResponse> | undefined,
+    doc: LogsEndpointAction
+  ): Promise<void> {
+    const createBaseDocument = (): LogsEndpointActionResponse => ({
+      '@timestamp': moment().toISOString(),
+      agent: doc.agent,
+      EndpointActions: {
+        action_id: doc.EndpointActions.action_id,
+        completed_at: moment().toISOString(),
+        started_at: moment().toISOString(),
+        data: doc.EndpointActions.data,
+        input_type: 'crowdstrike',
+      },
+    });
+
+    const document = createBaseDocument();
```

Comment:
This is confusing. ?you have a function (`createBaseDocument()` which is scoped to this class method (`completeCrowdstrikeAction()`) that returns static data (no logic and then you immediately call it.

Why do you even need `createBaseDocument()`??

Reply:
LOL, not sure why I left it like this, I used this createBaseDocument in two places before. 
Thanks for catching this!